src/examples/docker.yml

```diff
@@ -0,0 +1,29 @@
+description: |
+  This example shows how a docker workflow is organized.
+
+usage:
+  version: 2.1
+
+  orbs:
+    cyber4all: cyber4all/orb@2.0.0-alpha
+
+  workflows:
+    integrations:
+      jobs:
+        # tests the build
+        - cyber4all/publish:
+            deploy: false
+
+    delivery:
+      jobs:
+        # pushes staging to docker registry
+        - cyber4all/publish:
+            context: Docker
+            tag: staging
```

Comment:
Will this be an issue since we are only following the staging naming convention for competencies atm? CLARK and CARD releases would still look towards the latest tag